gradle/license.gradle

```diff
@@ -0,0 +1,14 @@
+apply plugin: 'com.github.hierynomus.license'
+
+license {
+    header = rootProject.file('HEADER.txt')
+    strictCheck = true
+    ignoreFailures = true
+    skipExistingHeaders = true
+    mapping {
+        java = 'SLASHSTAR_STYLE'
+        groovy = 'SLASHSTAR_STYLE'
+    }
+    ext.year = Calendar.getInstance().get(Calendar.YEAR)
```

Comment:
Actually, there should be the year of the last change.
I've skipped existing headers because every time the plugin will update all source files. It's very unnecessary.